mindsdb/interfaces/datastore/datastore.py

```diff
@@ -86,7 +86,7 @@ def save_datasource(self, name, source_type, source, file_path=None):
                 ,'kwargs': {}
             }
         elif source_type == 'clickhouse':
-            user = self.config['integrations']['default_clickhouse']['user']
+            user = self.config['integrations']['default_clickhouse']['user']            # FIXME
```

Comment:
`# FIXME`
unclear what should be fixed :))

Reply:
good notice :)